### ISSUE TITLE
Cannot download to directory forces /downloads

### ISSUE THREAD

**Issue by Vantageright** [axcore/tartube#28]
I want to download to Youtube. This puts things in Youtube/downloads/Whatever. How do I stop it.

**Comment by axcore** [comment]
@Vantageright, I haven't implemented that yet.

**Comment by 1024mb** [comment]
A workaround for this could be to create a symbolic link and rename it so you can foul Tartube. I haven't tested that so I'm just giving ideas.

**Comment by mbnoimi** [comment]
> A workaround for this could be to create a symbolic link and rename it so you can foul Tartube. I haven't tested that so I'm just giving ideas.

I do that since first release of Tartube and it's working fine to me but I believe Windows users will suffer because creating symbolic links is painful in Windows

**Comment by axcore** [comment]
I am probably going to implement this soon, so don't worry too much about symbolic links.

**Comment by axcore** [comment]
Implemented in v1.4.0. 

The data directory now looks like this:

```
/tartube-data
    /pewdiepie
    /T-series
    tartube.db
    /.backups
        tartube_BU.db
```

Tartube recognises both the old and new directory structures, so there is no need to move things around on your computer. If you do want to move things around, see the [README](https://github.com/axcore/tartube/blob/master/README.rst).

tl;dr: Find the `/tartube-data/downloads` directory (folder), move everything from that directory into the directory above, i.e. `/tartube-data`, and then delete the empty `/tartube-data/downloads` directory.
